src/main/java/com/github/onsdigital/babbage/search/helpers/SearchHelper.java

```diff
@@ -19,6 +23,9 @@
 
 public class SearchHelper {
 
+    // List of retired urls to be temporarily black-listed until content is cleared up (Trello card #482)
+    private static List<String> retiredUrls = Configuration.ELASTIC_SEARCH.getHighlightBlacklist();
```

Comment:
If you implement the singleton pattern you dont need to assign this to a class var you can just call get whenever you need it